python/cudf/cudf/core/column/string.py

```diff
@@ -4138,6 +4144,119 @@ def subword_tokenize(
             cupy.asarray(metadata),
         )
 
+    def porter_stemmer_measure(self, **kwargs):
+        """
+        Compute the Porter Stemmer measure for each string.
+        The Porter Stemmer algorithm is described `here
+        <https://tartarus.org/martin/PorterStemmer/def.txt>`_.
+
+        Returns
+        -------
+        Series or Index of object.
+
+        Examples
+        --------
+        >>> import cudf
+        >>> ser = cudf.Series(["hello", "super"])
+        >>> ser.str.porter_stemmer_measure()
+        0    1
+        1    2
+        dtype: int32
+        """
+        return self._return_or_inplace(
+            cpp_porter_stemmer_measure(self._column), **kwargs
+        )
+
+    def is_consonant(self, index, **kwargs):
```

Comment:
Can we name this something different than `index`? Index is already defined in Pandas / cuDF as a set of columns used for row accesses.

Reply:
Maybe we could name it `i` to be consistent with `get` : https://github.com/rapidsai/cudf/blob/branch-0.15/python/cudf/cudf/core/column/string.py#L1853-#L1860

Reply:
I'm ok with `i` 
Just a clarification, it can also be a column of integer values.

Reply:
I see, so `position` would then be more precise in that case?

Reply:
Sounds good to me.